moped-editor/src/components/NavLink.js

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { NavLink as BaseNavLink } from "react-router-dom";
+
+// Custom NavLink to preserve support for "activeClassName"
+// see: https://reactrouter.com/docs/en/v6/upgrading/v5#remove-activeclassname-and-activestyle-props-from-navlink-
+const NavLink = React.forwardRef(
+  ({ activeClassName, activeStyle, ...props }, ref) => {
+    return (
+      <BaseNavLink
+        ref={ref}
+        {...props}
+        className={({ isActive }) =>
+          [props.className, isActive ? activeClassName : null]
+            .filter(Boolean)
+            .join(" ")
+        }
+        style={({ isActive }) => ({
+          ...props.style,
+          ...(isActive ? activeStyle : null),
+        })}
+      />
+    );
+  }
+);
+
+export default NavLink;
```

Comment:
I tried a few different ways to deal with the [deprecated `activeClassName` prop](https://reactrouter.com/docs/en/v6/upgrading/v5#remove-activeclassname-and-activestyle-props-from-navlink-). 

Probably my own ineptitude, but this was the only one I got working. Also considered:
- updating our MUI classes to use an `active` pseudo-selector
- doing a global component override in the [MUI theme](https://mui.com/material-ui/guides/routing/#global-theme-link) 
- using a custom `className` function

Reply:
This makes sense to me, and it is an interesting problem. I like fixes like this that give kind of an under-the-hood view at what is going on.